unitTesting/test_bindings.py

```diff
@@ -29,6 +32,23 @@ def test_type_check(self):
             dg.plug(first, second.signal('in_double'))
         self.assertEqual(str(cm_out.exception), ERR % 'a')
 
+    def test_dg_exc(self):
+        """
+        test that exceptions from dynamic graph are correctly raised
+        """
+        ent = CustomEntity('test_dg_exc')
+        # check that accessing a non initialized signal raises
+        with self.assertRaises(dg.dgpyError) as cm:
+            ent.act()
+        self.assertEqual(
+            str(cm.exception),
+            'In SignalPtr: SIN ptr not set. (in signal <CustomEntity(test_dg_exc)::input(double)::in_double>)')
```

Comment:
@jmirabel : the exception is this one in this case. In your snipped for sot-core, it is the following: https://github.com/stack-of-tasks/sot-core/pull/118/files#diff-edb0f371937f31c9ee0e703d93c2ab48R15